developer-docs-site/docs/concepts/gas-txn-fee.md

```diff
@@ -181,3 +181,8 @@ The simulation steps for finding the correct amount of gas for a transaction are
 :::tip
 Prioritization is based upon buckets of `gas_unit_price`. The buckets are defined in [`mempool_config.rs`](https://github.com/aptos-labs/aptos-core/blob/30b385bf38d3dc8c4e8ee0ff045bc5d0d2f67a85/config/src/config/mempool_config.rs#L8). The current buckets are `[0, 150, 300, 500, 1000, 3000, 5000, 10000, 100000, 1000000]`. Therefore, a `gas_unit_price` of 150 and 299 would be prioritized nearly the same.
 :::
+
+## Gas Profiling
```

Comment:
there's redundancy with the section on

`## Estimating gas consumption via simulation`

would be good to unify